### PR TITLE
Governing body bugs Fixed #2268

### PR DESCRIPTION
**Pull request by Omeliukh-Nazar**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/GoverningBody/GoverningBody/GoverningBody.tsx [hunk 1]
```diff
@@ -196,23 +199,24 @@ const GoverningBody = () => {
     setLoading(true);
     try {
       let userAccesses = await getUserAccesses();
+      console.log(userAccesses);
```
Comment (VSBalanchuk) [inline src/pages/GoverningBody/GoverningBody/GoverningBody.tsx:202]: Prety sure, that you missed this one ;)

File: src/pages/GoverningBody/GoverningBody/GoverningBody.tsx [hunk 2]
```diff
@@ -196,23 +199,24 @@ const GoverningBody = () => {
...
         }
         setAnnouncements(shortListedAnnoncements)
       }
       const response = await getGoverningBodyById(+id);
       const governingBodyViewModel = response.data.governingBodyViewModel;
-
       const admins = [
-        ...governingBodyViewModel.administration,
         governingBodyViewModel.head,
+        ...governingBodyViewModel.administration
```
Comment (VSBalanchuk) [inline src/pages/GoverningBody/GoverningBody/GoverningBody.tsx:218]: Are you sure that is the correct order?

File: src/pages/GoverningBody/GoverningBody/GoverningBody.tsx (hunk 1 above)
Comment (VSBalanchuk) [inline src/pages/GoverningBody/GoverningBody/GoverningBody.tsx:202]: Pretty sure that you missed this one ;)

File: src/pages/GoverningBody/Sector/Sector.tsx [hunk 3]
```diff
@@ -130,12 +143,15 @@ const Sector = () => {
       const response = await getSectorById(+sectorId);
       const sectorViewModel = response.data.sectorViewModel
       let userAccesses = await getUserAccesses();
+      console.log(userAccesses);
```
Comment (VSBalanchuk) [inline src/pages/GoverningBody/Sector/Sector.tsx:146]: And this one.

File: src/pages/GoverningBody/GoverningBody/GoverningBody.tsx (hunk 1 above)
Comment (Omeliukh-Nazar) [reply src/pages/GoverningBody/GoverningBody/GoverningBody.tsx]: Thanks for this)

**Review by DimaDiD — APPROVED** [review]

**Review by VSBalanchuk — APPROVED** [review]

**Pull request merged**